Zend/zend_compile.c

```diff
@@ -4873,7 +4909,28 @@ void zend_compile_func_decl(znode *result, zend_ast *ast) /* {{{ */
 
 	zend_compile_params(params_ast, return_type_ast);
 	if (uses_ast) {
-		zend_compile_closure_uses(uses_ast);
+		uint32_t i;
```

Comment:
Is it possible to export this to another function? Growing and growing existing functions is a path to nightmare.